jbpm-services/jbpm-services-api/src/build/revapi-config.json

```diff
@@ -17,7 +17,82 @@
     "ignores": {
         "revapi": {
             "_comment": "Changes between 7.44.0.Final and the current branch. These changes are desired and thus ignored.",
-            "ignore": []
+            "ignore": [
+                {
+                    "code": "java.method.addedToInterface",
+                    "new": "method java.lang.String org.jbpm.services.api.model.UserTaskInstanceDesc::getCorrelationKey()",
+                    "package": "org.jbpm.services.api.model",
+                    "classSimpleName": "UserTaskInstanceDesc",
+                    "methodName": "getCorrelationKey",
+                    "elementKind": "method",
+                    "justification": "https://issues.redhat.com/browse/BAPL-1623"
+                },
+                {
+                    "code": "java.method.addedToInterface",
+                    "new": "method void org.jbpm.services.api.model.UserTaskInstanceDesc::setCorrelationKey(java.lang.String)",
+                    "package": "org.jbpm.services.api.model",
+                    "classSimpleName": "UserTaskInstanceDesc",
+                    "methodName": "setCorrelationKey",
+                    "elementKind": "method",
+                    "justification": "https://issues.redhat.com/browse/BAPL-1623"
+                },
+                {
+                    "code": "java.method.addedToInterface",
+                    "new": "method java.lang.Integer org.jbpm.services.api.model.UserTaskInstanceDesc::getProcessType()",
+                    "package": "org.jbpm.services.api.model",
+                    "classSimpleName": "UserTaskInstanceDesc",
+                    "methodName": "getProcessType",
+                    "elementKind": "method",
+                    "justification": "https://issues.redhat.com/browse/BAPL-1623"
+                },
+                {
+                    "code": "java.method.addedToInterface",
+                    "new": "method void org.jbpm.services.api.model.UserTaskInstanceDesc::setProcessType(java.lang.Integer)",
+                    "package": "org.jbpm.services.api.model",
+                    "classSimpleName": "UserTaskInstanceDesc",
+                    "methodName": "setProcessType",
+                    "elementKind": "method",
+                    "justification": "https://issues.redhat.com/browse/BAPL-1623"
+                },
```

Comment:
This ignore section is not needed. These changes are already in 7.44 final version.

Reply:
ok